cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/transforms/IdentityTransform.java

```diff
@@ -16,23 +16,23 @@
 
 package co.cask.cdap.templates.etl.transforms;
 
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
 import co.cask.cdap.templates.etl.api.Emitter;
-import co.cask.cdap.templates.etl.api.StageConfigurer;
 import co.cask.cdap.templates.etl.api.TransformStage;
 
 /**
- * Simple Identity Transform for testing.
+ * Simple Identity Transform.
  *
  * @param <T> any type
  */
+@Plugin(type = "transform")
+@Name("IdentityTransform")
```

Comment:
Actually, can we just call it `Identity`, because the plugin already tells it's "transform" ? What should be the naming convention?